program-runtime/src/cpi.rs

```diff
@@ -159,7 +162,12 @@ fn check_account_infos(
 ) -> Result<(), Error> {
     let max_cpi_account_infos = if invoke_context
         .get_feature_set()
-        .increase_tx_account_lock_limit
+        .increase_cpi_account_info_limit
+    {
+        MAX_CPI_ACCOUNT_INFOS_SIMD_0339
+    } else if invoke_context
+            .get_feature_set()
+            .increase_tx_account_lock_limit
     {
         MAX_CPI_ACCOUNT_INFOS
     } else {
```

Comment:
These two feature gates conflict with each other. Is SIMD-0339 supposed to supersede `increase_tx_account_lock_limit` ?

Reply:
Yes, SIMD-0339 will override the `increase_tx_account_lock_limit` feature here